python/activator/middleware_interface.py

```diff
@@ -721,15 +740,15 @@ def _export_calib_associations(self, calib_collection, datasets):
         # latest calibs as of today. Already-cached calibs are still available
         # from previous collections.
         if datasets:
-            calib_daily = f"{calib_collection}/{self._day_obs}"
-            new = self.butler.registry.registerCollection(calib_daily, CollectionType.CALIBRATION)
+            calib_latest = self._get_local_calib_collection(calib_collection)
+            new = self.butler.registry.registerCollection(calib_latest, CollectionType.CALIBRATION)
             if new:
```

Comment:
It shouldn't given the "always create a new collection" implementation, but keeping it felt safer than allowing existing members of the chain to be rearranged (which would break invariants like "the front of the chain is always the latest").

In any case, I'm going back to daily calibs, so this check is still useful.